api/nodecontrol/client.go

```diff
@@ -122,7 +107,7 @@ func (c *ControlAPIClient) DirectPing(nodeId string) (*nodegen.NodePingResponseJ
 }
 
 func (c *ControlAPIClient) FindWorkload(_type, namespace, workloadId string) (*nodegen.WorkloadPingResponseJson, error) {
-	msg, err := c.nc.Request(models.AgentPingWorkloadRequestSubject(_type, namespace, workloadId), nil, DefaultRequestTimeout)
+	msg, err := c.nc.Request(models.WorkloadPingRequestSubject(_type, namespace, workloadId), nil, DefaultRequestTimeout)
```

Comment:
If we're actually using this variable, we should remove the `_` ... Weird that the linter doesn't complain about this when it's so cranky about the inverse.

Reply:
its because `type` is a reserved word.....I can call it something else

Reply:
oh, hah.. makes sens. yeah, maybe `workloadType`